align_system/algorithms/random_adm.py

```diff
@@ -1,6 +1,6 @@
 import random
 
-from swagger_client.models import ActionTypeEnum, InjuryLocationEnum, CharacterTagEnum
+from swagger_client.models import ActionTypeEnum, InjuryLocation, CharacterTagEnum
```

Comment:
@dmjoy, do you know what the guidance is on `InjuryLocationEnum` vs `InjuryLocation`? I was running into errors when `InjuryLocationEnum` was being used, but that looks like what we're using in the single KDMA ADM

Reply:
I'm not certain, it looks like in the server code they're exactly the same apart from the name.. what error were you running into?

Reply:
`InjuryLocation` has less options than `InjuryLocationEnum` (and they seem to be validating against `InjuryLocation`), so I'd get an error when it chose an "invalid" location from `InjuryLocationEnum`

Reply:
Ah, it looks like they've made some changes to the `development` branch of the client (and specifically the `InjuryLocationEnum` class) in the last few days.  Can you install the `main` branch of the client library instead?  I'll update the `pyproject.toml` file to this effect as well, but at the time that was THE branch to be using as we approached the metrics evaluation, but now `main` is the one we should target